<launch>


<group ns="rosbag">
    <node pkg="rosbag" type="play" name="replay" args="--clock -l /home/thomas/Documenti/MasterThesis/bagfiles/marker_calibration/marker_ir_test.bag"/>
</group>


<group ns="marker_finder">
    <arg name="queue"        default="1000"  />
    <arg name="inTopic"     default="/k01/ir/image_rect" /> <!-- /k01/ir/image_rect  /kinect2/sd/image_ir_rect-->
    <arg name="inInfo"      default="/k01/ir/camera_info"/>
    <arg name="outTopic"    default="image" />

    <node pkg="marker_finder" type="node" name="finder" output="screen">
        <param name="queue"           value="$(arg queue)"            />
        <param name="outTopic"       value="$(arg outTopic)"        />
        <param name="inTopic"        value="$(arg inTopic)"         />

    </node>
</group>

<node pkg="rqt_image_view" type="rqt_image_view" name="image_view"/>

</launch>